crates/bitcoin-da/src/lib.rs

```diff
@@ -68,10 +71,19 @@ impl fmt::Display for BitcoinError {
             BitcoinError::ReadNoDataErr => write!(f, "Read no data in tx error"),
             BitcoinError::GetBlockErr => write!(f, "Get block error"),
             BitcoinError::GetBlockchainInfoErr => write!(f, "Get blockchain info error"),
+            BitcoinError::RpcError(ref e) => write!(f, "RPC error: {}", e),  // New match arm
```

Comment:
This is SO USEFULL thanks, idk why I didn't implement it before

Reply:
No problem! Glad it helps